contract/Cargo.toml

```diff
@@ -7,15 +7,10 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
```

Comment:
I have noticed that `tests/wasm/contract.wasm` is included into the repository. Please remove it and add it to `.gitignore`.

Reply:
I added it to .gitignore

Reply:
Sorry, misread it as `target` instead of `target`